lib/screens/player_screen.dart

```diff
@@ -94,10 +100,19 @@ class _PlayerScreenContent extends StatelessWidget {
           }
         }
       },
+      onHorizontalSwipe: (direction) {
+        if (direction == SwipeDirection.left) {
```

Comment:
I guess we can. My original reasoning was that this would be a way to get to the lyrics screen even if lyrics can't be loaded, because that screen shows more info about why they couldn't be loaded.  
But a hidden swipe gesture is probably not the right approach for this.

I guess this would mean refactoring the `_PlayerScreenContent` to be a `ConsumerWidget`?

The other option we have would be to not fully disable the button, instead gray it out and change the icon if lyrics can't be loaded, but still letting users tap on it to see the reason why the lyrics aren't available. Disabled elements are "bad" UX-wise anyway...  
What do you think about that?

Reply:
Greying the button out but still letting people tap through to the lyrics screen seems pretty reasonable, and doing it with the button seems more reasonable than only having the swipe.  It's also harder to accidentally trigger, which is the main reason I want the swipe disabled when we don't have lyrics.

Reply:
So you want both changes? Making the button not fully disabled, and also converting to a ConsumerWidget so disable the swipe gesture when lyrics are missing?

Reply:
That would be my preferred setup, yeah.

Reply:
Your wish has been granted ✨